.github/workflows/pmm-package-test-matrix-full.yml

```diff
@@ -0,0 +1,336 @@
+      metrics_mode: ${{ inputs.metrics_mode || 'auto'}}
+      playbook: "pmm2-client_integration_auth_register"
+      test_name: ${{ matrix.os }}
+      os: ${{ matrix.os }}
+
+  auth_setup_test:
+    name: "pmm2-client_integration_auth_setup"
+    uses: ./.github/workflows/pmm-package-test-runner.yml
+    secrets: inherit
+    needs: get_versions
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["bionic-x64",
+             "bullseye-x64",
+             "buster-x64",
+             "focal-x64",
+             "jammy-x64",
+             "centos-7-x64",
+             "ol-8-x64",
+             "ol-9-x64"]
+    with:
+      package_testing_branch: ${{ inputs.package_testing_branch || 'master' }}
+      expected_version: ${{needs.get_versions.outputs.finish_version}}
+      repository: ${{ inputs.repository || 'release'}}
+      metrics_mode: ${{ inputs.metrics_mode || 'auto'}}
+      playbook: "pmm2-client_integration_auth_setup"
+      test_name: ${{ matrix.os }}
+      os: ${{ matrix.os }}
+
+  tbi_test:
+    name: "pmm2-client_integration_custom_path"
+    uses: ./.github/workflows/pmm-package-test-runner.yml
+    secrets: inherit
+    needs: get_versions
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["bionic-x64",
+             "bullseye-x64",
+             "buster-x64",
+             "focal-x64",
+             "jammy-x64",
+             "centos-7-x64",
+             "ol-8-x64",
+             "ol-9-x64"]
+    with:
+      package_testing_branch: ${{ inputs.package_testing_branch || 'master' }}
+      expected_version: ${{needs.get_versions.outputs.finish_version}}
+      repository: ${{ inputs.repository || 'release'}}
+      metrics_mode: ${{ inputs.metrics_mode || 'auto'}}
+      playbook: "pmm2-client_integration_custom_path"
+      test_name: ${{ matrix.os }}
+      os: ${{ matrix.os }}
+
+  port_test:
+    name: "pmm2-client_integration_custom_port"
+    uses: ./.github/workflows/pmm-package-test-runner.yml
+    secrets: inherit
+    needs: get_versions
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["bionic-x64",
+             "bullseye-x64",
+             "buster-x64",
+             "focal-x64",
+             "jammy-x64",
+             "centos-7-x64",
+             "ol-8-x64",
+             "ol-9-x64"]
+    with:
+      package_testing_branch: ${{ inputs.package_testing_branch || 'master' }}
+      expected_version: ${{needs.get_versions.outputs.finish_version}}
+      repository: ${{ inputs.repository || 'release'}}
+      metrics_mode: ${{ inputs.metrics_mode || 'auto'}}
+      playbook: "pmm2-client_integration_custom_port"
+      test_name: ${{ matrix.os }}
+      os: ${{ matrix.os }}
+
+  port_u_test:
+    name: "pmm2-client_integration_upgrade_custom_port"
+    uses: ./.github/workflows/pmm-package-test-runner.yml
+    secrets: inherit
+    needs: get_versions
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["bionic-x64",
```

Comment:
@vyarosh could you replace this with the list of OS you added

Reply:
I need this one merged to have comprehensive testing from UI